### PR TITLE
Change file header

### PR DESCRIPTION
**Pull request by robertmain**
Switches file header from using `//!` to using `/*!` followed by `*`.

### CODE REVIEW

File: src/moment.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-//! authors : Tim Wood, Iskren Chernev, Moment.js contributors
-//! license : MIT
-//! momentjs.com
+/*!
+* moment.js
+* version : 2.10.6
+* authors : Tim Wood, Iskren Chernev, Moment.js contributors
+* license : MIT
+* momentjs.com
+*/!
```
Comment (icambron) [inline src/moment.js]: That exclamation mark is outside the comment, so it's not valid JS.

**Comment by icambron** [comment]
What is the goal here?

File: src/moment.js (hunk 1 above)
Comment (robertmain) [inline src/moment.js]: Well spotted. I'll fix that.:thumbsup:

**Comment by robertmain** [comment]
The goal was to have momentjs' license header in a similar format to most other js libraries. 

https://code.google.com/p/share-extras/wiki/LicenseTemplate

**Comment by ichernev** [comment]
Still not convinced

**Comment by robertmain** [comment]
It just becomes a little more readable when you run it through minifiers and the like but okay...whatever

**Pull request closed without merging**